README.md

```diff
@@ -39,3 +39,18 @@ bdd_tests            Run BDD tests
 before_commit        Checks done before commit
 help                 Show this help screen
 ```
+
+## Database
+
+PostgreSQL database is used as a storage.
+
+Please look [at detailed schema
+description](https://redhatinsights.github.io/ccx-notification-service/db-description/)
+for more details about tables, indexes, and keys.
+
+## Schema description
+
+DB schema description can be generated by `generate_db_schema_doc.sh` script.
+Output is written into directory `docs/db-description/`. Its content can be
+viewed [at this
+address](https://redhatinsights.github.io/ccx-notification-writer/db-description/).
```

Comment:
I know the DB is the same, but just in case, did you mean to write 'ccx-notification-writer' here?

Reply:
actually nope, I'll fix it, thanks